src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -787,14 +456,14 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
             <Box textAlign='right'>
               <Amount.Fiat
                 fontWeight='bold'
-                value={bnOrZero(networkFeeCryptoBaseUnit)
+                value={bnOrZero(estimatedFeesData?.estimatedFees.fast.txFee)
```

Comment:
can use `fromBaseUnit` here and below

Reply:
also gm c3937b3de53e37821bf9f9bec7ebd6c5f15df04b